tests/covalent_dispatcher_tests/_core/execution_test.py

```diff
@@ -294,17 +318,15 @@ def workflow(x):
     assert result_object._error == "The following tasks failed:\n0: failing_task"
 
 
-async def test_run_workflow_does_not_deserialize(mocker):
+@pytest.mark.asyncio
+async def test_run_workflow_does_not_deserialize(mocker, test_cluster, event_loop):
     """Check that dispatcher does not deserialize user data when using
     out-of-process `workflow_executor`"""
 
-    from dask.distributed import LocalCluster
-
     from covalent._workflow.lattice import Lattice
     from covalent.executor import DaskExecutor
```

Comment:
Let's also remove these from here, we're already importing `Lattice` at the top of the file.

Reply:
Done...